default.yaml

```diff
@@ -0,0 +1,12 @@
+repos:
+  dummy-app:
```

Comment:
we should put this as example.yaml.  also, if configPath doesnt exist, does it fail?